en/telemetry/crossfire_telemetry.md

```diff
@@ -0,0 +1,165 @@
+
+There is no need to set the baud rate for the port, as this is configured by the driver.
+
+## Radio Setup
+
+See [TBS Crossfire Manual](https://www.team-blacksheep.com/tbs-crossfire-manual.pdf) (search for 'Setting up radio for CRSF').
+
+<a id="transmitters"></a>
+
+## Compatible RC Transmitters
+
+:::note
+An RC radio system historically consisted of a handheld controller on the ground that transmitted to an on-vehicle receiver.
+Even though many radio systems are now bidirectional, the ground module is often referred to as the transmitter, and the air unit is called a receiver.
+:::
+
+You will need an RC transmitter that can receive the CRSF telemetry stream from a bound CRSF receiver on the vehicle.
+
+Radio controllers may come with an inbuilt base module ("transmitter") [such as these by TBS](https://www.team-blacksheep.com/shop/q:radio%20drone%20controller)
+
+Other ground controllers allow transmitters to be added as separate transmitter modules.
+TBS transmitters are usually compatible with the "JR module bay".
+For example the [FrSky Taranis X9D Plus (2)](https://www.frsky-rc.com/product/taranis-x9d-plus-2/) can be used with TBS modules
+
+Popular alternatives that can be used with [TBS Transmitter Modules](http://team-blacksheep.com/shop/cat:rc_transmitters#product_listing) include:
+
+- FrSky Taranis X9D Plus (recommended)
+- ?
+
+These transmitters will need to be configured for use with your transmitter.
+The [TBS Crossfire Manual](https://www.team-blacksheep.com/tbs-crossfire-manual.pdf) provides some common setup instructions (i.e. for OpenTX running on the Taranis X9D plus).
+
+<a id="receivers"></a>
+
+## Telemetry Receivers
+
+<!-- what should we list here? -->
+
+- [TBS Crossfire Nano RX](http://team-blacksheep.com/products/prod:crossfire_nano_rx) - designed for small Quads.
+- ELRS RX <!-- add link and more detail -->
+
+## Wiring
+
+The TX and RX on your selected Flight Controller UART should be connected to separate channels on the receiver.
+If using a TBS receiver the signal is uninverted, and can be directly connected (no additional inverter logic is required in the cable).
+
+<!-- ELRS - uninverted too? Is setup the same? -->
+<!-- how do I set up the channels that are used to mean tx/rx. The receivers I am seeing appear to have 4 channels on them. What's the story? Are these supposed to be single wire? --->
+
+<!--
+Connect the Nano RX and Omnibus pins as shown:
+
+| Omnibus UART1 | Nano RX |
+| ------------- | ------- |
+| TX            | Ch2     |
+| RX            | Ch1     |
+
+-->
+
+### Pixhawk
+
+<a id="messages"></a>
+
+## Telemetry Messages
+
+The [TBS Crossfire Manual](https://www.team-blacksheep.com/tbs-crossfire-manual.pdf) includes a list of messages: search on "Available sensors with OpenTX".
```

Comment:
@chris1seto 
- Do we support all the telemetry in the linked table? (in which case I'll copy it in). 
- I presume when it says for OpenTX we basically can work with CRSF on any system that supports OpenTX and can connect to module? Do we know what that set is?